src/simularium/VisData.ts

```diff
@@ -63,7 +62,9 @@ class VisData {
             };
 
             for (let k = 0; k < AGENT_OBJECT_KEYS.length; ++k) {
-                agentData[AGENT_OBJECT_KEYS[k]] = floatView[j++];
+                const key = AGENT_OBJECT_KEYS[k];
+                const targetKey = key === "vis-type" ? "visType" : key;
```

Comment:
Question: does simulariumio output `vis-type` anymore?  Will we ever see data with this key name? I am sort of hoping we can remove it forever.  Also I suggest putting the `if key is vis-type then rename it to visType` into a single function so that `vis-type` is not repeated in more than one place.  The function can have a pointed name like `fixLegacyBadAgentPropertyName` or something...

Reply:
@ascibisz what do you think about this?

Reply:
Simulariumio doesn't output anything with `vis-type` or `visType` in it, as it doesn't set key names in that part of the data at all, the type of data each number represents is assumed based on its position. So as long as all references to `vis-type` are changed to `visType` within this repo, you should be fine. I don't see any references to `vis-type` anywhere else.

Reply:
I see, so even when we stream a json trajectory, the agent data is just in an ordered array of numbers without key names..?  Which means we can totally replace "vis-type" throughout this code!

Reply:
this line should probably change: https://github.com/simularium/octopus/blob/e4ef3da8b2c1f0ed1d01125194fd778a352cfa62/octopus/agent.py#L23

Reply:
Correct! I will change that line of code, to be safe, although it's not currently getting called anywhere in Octopus